packages/dev/core/src/Meshes/Node/Blocks/randomBlock.ts

```diff
@@ -163,6 +166,9 @@ export class RandomBlock extends NodeGeometryBlock {
                     case RandomBlockLocks.LoopID:
                         lockId = state.getContextualValue(NodeGeometryContextualSources.LoopID, true) || 0;
                         break;
+                    case RandomBlockLocks.Once:
+                        lockId = state.buildId || 0;

```

Comment:
Shouldn't the `|| 0` be a no-op since `buildId` is already a non-nullable number? Or is this possible because we don't enable the 'strictPropertyInitialization` compiler flag?

Reply:
it is more when users mess up with our data from the outside (in JS)